src/viam/examples/project/cmake/CMakeLists.txt

```diff
@@ -41,9 +41,9 @@ find_package(Threads REQUIRED)
 # In practice, you should set a minimum required version
 # when loading the SDK:
 #
-# find_package(viam-cpp-sdk 1.0.2 CONFIG_REQUIRED COMPONENTS Lib)
+# find_package(viam-cpp-sdk 1.0.2 CONFIG REQUIRED COMPONENTS viamsdk)
 #
-find_package(viam-cpp-sdk CONFIG REQUIRED COMPONENTS Lib)
+find_package(viam-cpp-sdk CONFIG REQUIRED viamsdk)
```

Comment:
Before, we only had a component named `Lib`. Now we have two components `viamapi` and `viamsdk` for the API library and SDK library respectively.